timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.kt

```diff
@@ -65,6 +66,7 @@ class WrongTimberUsageDetectorTest {
                 |  }
                 |}""".trimMargin())
         )
+        .allowMissingSdk()
```

Comment:
should we be allowing missing sdk?

Reply:
I think I set this because running tests in Android Studio was having issues picking up my `ANDROID_HOME` and all the tests still worked after setting it.